src/app/item-page/simple/field-components/specific-field/abstract/item-page-abstract-field.component.ts

```diff
@@ -36,4 +36,5 @@ export class ItemPageAbstractFieldComponent extends ItemPageFieldComponent {
      */
     label = 'item.page.abstract';
 
+    enableMarkdown = true;
```

Comment:
Please add a comment here to describe what this is doing.  It appears it enables *both* Markdown and MathJax, as long as they are enabled in your `config.*.yml`?

Reply:
I've added some extra explanation here